services/api-server/src/simcore_service_api_server/core/application.py

```diff
@@ -5,10 +5,16 @@
 from models_library.basic_types import BootModeEnum
 from packaging.version import Version
 from servicelib.fastapi.profiler_middleware import ProfilerMiddleware
-from servicelib.fastapi.tracing import setup_tracing
+from servicelib.fastapi.tracing import (
+    setup_tracing,
+    setup_tracing_spans_for_simcore_service_functions,
+)
 from servicelib.logging_utils import config_all_loggers
+from simcore_service_api_server import exceptions
+
+setup_tracing_spans_for_simcore_service_functions()
```

Comment:
I really do not like that the setup has to be called depending on the imports
if it is experimental, why not to use DEV_FEATURE to enable/disable it?
I assume you will be on top of it right?